### DISCUSSION TITLE
Command-line equivalents of `test.` scoped using directives?

### DISCUSSION THREAD

**Discussion by SethTisue** [VirtusLab/scala-cli#2204] [General]
For example, `using resourceDir` has the command-line equivalent `--resource-dir`.

What is the command-line equivalent of `using test.resourceDir`?

These don't work:

```
% scala-cli test --test.resource-dir foo . 
Unrecognized argument: --test.resource-dir
% scala-cli test --test-resource-dir foo .
Unrecognized argument: --test-resource-dir
% scala-cli test --test --resource-dir foo .
Unrecognized argument: --test
```

Feel free to convert the discussion to an issue if there isn't any syntax for this :-)

**Comment by Gedochao** [comment]
We don't have a command-line equivalent right now, directives only.
Overall, implementing this is a rather large effort, as there's quite a bunch of things we'd need to refactor to achieve that...

**Comment by lwronski** [comment]
@SethTisue, what use case do you have that is not covered by `scala-cli test --resource-dir foo .`?

**Reply by SethTisue** [reply]
I don't have one. I was testing something and I didn't strictly _need_ the resources to only be on the test classpath and not the main classpath. So for my own purposes, I'm satisfied to simply have it confirmed that no command line equivalent exists. We could wait and see if anyone else asks or cares.
